postprocessing/OEP_MaStR_cleansing.ipynb

```diff
@@ -36,14 +36,21 @@
    },
```

Comment:
@BenPortner - is it necessary to have the outputs of the notebook in the repo? If not, could you push a version without the outputs?

Reply:
Hi @Bachibouzouk! The branch used for the PR should not contain any outputs. Let me know if I'm wrong

Reply:
Or did you mean the notebook output cells? Do you want me to remove them all or only the graphics?

Reply:
Yes I meant the notebook output cells ^^

I trust you to know what should not be removed. If the purpose of this notebook is mainly to be used locally by other users, I would remove the notebook's output cells.

Reply:
I'll remove the graphics to make the notebook smaller. I'll keep the df heads and numeral outputs. Sound good?

Reply:
> 
> 
> @BenPortner - is it necessary to have the outputs of the notebook in the repo? If not, could you push a version without the outputs?

I'm using conda with channel conda-forge. Not sure it harmonizes well with the requirements.txt. I'll add an environment.yml instead. Will that work?

Reply:
Decided to remove all outputs after all!

Reply:
I approved the PR, so you can merge whenever you want to :)

Reply:
I would like to merge, but I think I have no write access...

> Only those with write access to this repository can merge pull requests.